src/main/java/com/sekwah/advancedportals/bukkit/listeners/Listeners.java

```diff
@@ -48,6 +50,10 @@ public Listeners(AdvancedPortalsPlugin plugin) {
         }
 
         plugin.getServer().getPluginManager().registerEvents(this, plugin);
+
+        int cleanPeriod = config.getConfig().getInt("CleanUpPeriod", 120);
```

Comment:
Add new config field "CleanUpPeriod" so admins can define how often (in minutes) cleanup should be processed, default is 2h